dev/com.ibm.ws.rest.handler.validator_fat/test-applications/testOpenAPIApp/resources/META-INF/openapi.yaml

```diff
@@ -16,12 +16,29 @@ info:
 servers:
   - url: https://127.0.0.1:8020/ibm/api
 paths:
+  /config/:
+    get:
+      tags:
+      - Config
+      summary: "Shows configuration of all elements"
+      description: "Retrieves configuration detail for instances of all configuration element types."
+      parameters:
+        - $ref: "#/components/parameters/queryParams"
+      responses:
+        200:
+          description: "Configuration info retrieved"
+          content:
+            application/json:
+              schema:
+                type: array
+                items:
+                  $ref: "#/components/schemas/config.result"
   /config/{elementName}:
```

Comment:
I know it's not related to these changes, but would `{elementType}` be better than `{elementName}`?

Reply:
I'm not really sure which way to go with the terminology.
I could see someone saying that the type of configuration element that you put in server config is `dataSource` or `connectionManager`.  I could also see someone saying that the type of configuration that you put in server config is a data source or a connection manager, and that the name of these configuration elements that you put into server config is `dataSource` and `connectionManager`.  I suppose it would be best to go with whichever way the majority see it, but I'm unsure what that is.